app/assets/javascripts/switch.js

```diff
@@ -0,0 +1,30 @@
+(function($) {
+  $.fn.switch = function() {
```

Comment:
I think we should start giving a namespace to these custom jquery functions.

`mu` from `MUdamos`

`$.fn.muSwitch`

or anything else you come up with.

Namespace (or prefix for that matter) avoid plugin collisions.